server/middleware/forceHTTPS.js

```diff
@@ -0,0 +1,12 @@
+function forceHTTPS(req, res, next) {
+
+    if (process.env.FORCE_HTTPS === 'true' && !req.secure) {
+       return res.redirect("https://" + req.headers.host + req.url);
+    }
+
+    next();
+}
+
+module.exports = {
+    forceHTTPS
```

Comment:
nit: airbnb styleguide suggests having commas at the ends of all object params and semi-colon at the end of the variable assignment. This is what it would look like altogether:
```
module.exports = {
  forceHTTPs,
};
```